miso-web/src/main/resources/security.properties

```diff
@@ -38,6 +38,12 @@ security.ldap.stripRolePrefix:MISO_
 
 security.ad.emailDomain:your_email_domain
 security.ad.url:ldap://your.active_directory.server:389
+# Normally, the email domain can be converted to the base DN for a user. So, if
+# your email domain is example.com, then the inferred domain DN will be
+# dc=example,dc=com. If your AD is set up strangely where this is not the case,
+# override the base DN in which the query for
+# userPrinicpalName={username}@{emailDomain} will occur.
+security.ad.domainDn:
```

Comment:
Can you give an example of what a correct value would look like?

Reply:
Not really. You have to know it from your LDAP server.